cmake/checks/check_01_cpu_features.cmake

```diff
@@ -173,6 +176,9 @@ IF(DEAL_II_ALLOW_PLATFORM_INTROSPECTION)
 
   CHECK_CXX_SOURCE_RUNS(
     "
+    #ifndef __AVX512F__
+    #error \"__AVX512F__ flag not set, no support for AVX512\"
```

Comment:
Out of curiosity, why do you need to escape the quotes here? You don't do so in other places.

Reply:
Because otherwise cmake interprets the first `"` as the closing quote of the first argument of `CMAKE_CXX_SOURCE_RUNS`.

Reply:
Oh, duh!

Thanks for pointing out the obvious!